server/jamovi/core.pyx

```diff
@@ -564,6 +588,86 @@ cdef class Column:
 
         self._this.setLevels(new_levels)
 
+    def parse_missing_value(self, missing_value):
+        optr = 0
+        new_value = missing_value.strip()
+        if new_value.startswith('=='):
+            optr = 0
+            new_value = new_value[2:].strip()
+        elif new_value.startswith('!='):
+            optr = 1
+            new_value = new_value[2:].strip()
+        elif new_value.startswith('<='):
+            optr = 2
+            new_value = new_value[2:].strip()
+        elif new_value.startswith('>='):
+            optr = 3
+            new_value = new_value[2:].strip()
+        elif new_value.startswith('<'):
+            optr = 4
+            new_value = new_value[1:].strip()
+        elif new_value.startswith('>'):
+            optr = 5
+            new_value = new_value[1:].strip()
+
+        type = 0
+        if new_value.startswith('"') or new_value.startswith("'"):
+            type = 0
+            new_value = new_value[1:-1]
+        else:
+            type = 1
+            new_value = float(new_value)
+            if new_value.is_integer():
```

Comment:
you need the bounds check too:

```
if new_value.is_integer() and new_value <= 2147483647 and new_value >= -2147483648:
```

Reply:
there's some functions here you can use:

https://github.com/jamovi/jamovi/blob/master/server/jamovi/server/utils/__init__.py#L11-L21